formats.js

```diff
@@ -1,6 +1,7 @@
 var moment = require('moment');
 var format = require('util').format;
 var validUrl = require('valid-url');
+const NUMBER_OF_ALLOWED_DECIMALS = 3;
```

Comment:
* Maybe move it to here like the other related fields? https://github.com/debitoor/schemagic/blob/f821dd638d4b3c4e4bf9c7a3f9aa33b8db5eb107/formats.js#L57
* Checked code now, maybe something like `currencyDecimalsLimit` (or something else with `currency` in name?)

Just thoughts.

Reply:
I think `CURRENCY_DECIMALS_LIMIT` can be ok